css/main.css

```diff
@@ -1,3 +1,733 @@
+}
+
+.feature-img img {
+  width: 100%;
+  height: 30rem;
+  object-position: center;
+  object-fit: cover;
+  position: relative;
+}
+
+.feature-title-btn {
+  position: absolute;
+  width: 100%;
+  padding: 1rem 0;
+  z-index: 1;
+}
+
+.feature-title-btn h2 {
+  margin-bottom: 0;
+  letter-spacing: 0.1rem;
+}
+
+.border-bottom-h2 {
+  width: 1rem;
+  background-color: var(--red_brown);
+  padding: 0.1rem 0.2rem;
+  margin: 1rem auto;
+}
+
+.about {
+  font-size: 1rem;
+  line-height: 1.7rem;
+}
+
+.about-us {
+  font-weight: 100;
+  font-size: 1.2rem;
+  letter-spacing: 0.7rem;
+  padding: 0;
+}
+
+.about-us-text {
+  padding: 4rem 1rem;
+}
+
+.img-about .img-area-about,
+.img-about .img-leaf-about {
+  width: 100%;
+  margin: -1rem 0;
+}
+
+.offer .offer-img {
+  width: 100%;
+  margin: -1.8rem 0;
+}
+
+.offer-online {
+  height: 25rem;
+  background-color: var(--green_black);
+  color: var(--white);
+}
+
+.offer-online .offer-online-h1 {
+  font-size: 3.2rem;
+  margin: -0.2rem 0;
+  padding-top: 4rem;
+}
+
+.offer-btn,
+.footer-subscribe-btn {
+  background-color: var(--white);
+  color: var(--green_black);
+}
+
+.offer-btn {
+  margin-top: 3rem;
+  padding: 0.9rem 3rem;
+}
+
+.form {
+  background-color: var(--off_white);
+  display: flex;
+  flex-direction: column;
+  padding: 0 1rem;
+}
+
+.form h3 {
+  font-size: 1.2rem;
+  font-weight: 100;
+  padding: 0 3rem;
+  letter-spacing: 0.4rem;
+}
+
+.form-section,
+.name {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  text-align: left;
+  color: #000;
+}
+
+.form-section label {
+  padding: 1rem 0;
+  font-size: 1.3rem;
+}
+
+.form-section input {
+  padding: 1rem 0;
+}
+
+.form-section input,
+.form-section textarea {
+  border: 1px solid #000;
+}
+
+.form-section input:hover,
+.form-section textarea:hover {
+  border: 3px solid #000;
+}
+
+.form-btn {
+  margin-top: 1rem;
+  margin-bottom: 2.5rem;
+}
+
+.img-last-section {
+  width: 100%;
+  margin: -2px 0;
+}
+
+.footer-green-background {
+  grid-area: footer;
+  background-color: var(--green_black);
+  color: var(--white);
+  text-align: center;
+  margin: -5px 0;
+}
+
+footer section {
+  padding-top: 70px;
+}
+
+.footer-our-store h3,
+.footer-hours h3,
+.footer-help h3,
+.footer-subscribe h3 {
+  font-size: 1.3rem;
+  letter-spacing: 0.4rem;
+  font-weight: initial;
+}
+
+.footer-our-store p,
+.footer-hours p {
+  margin: 0.2rem;
+}
+
+.footer-our-store p,
+.footer-hours p,
+.footer-help li a {
+  font-size: 1.2rem;
+}
+
+.footer-help ul {
+  list-style: none;
+  padding: 0;
+}
+
+.footer-help li {
+  margin-top: 0.4rem;
+}
+
+.footer-help li a {
+  text-decoration: underline;
+  color: var(--white);
+}
+
+.footer-subscribe .subscribe {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  text-align: left;
+  padding: 0 1rem;
+}
+
+.footer-subscribe label {
+  font-size: 1.3rem;
+  margin-bottom: 0.5rem;
+}
+
+.footer-subscribe input {
+  padding: 15px 6.6rem;
```

Comment:
This is too much padding, it makes the inner text start too far inside. 
For inputs, padding is used just to create a little space between the text and the edges, and perhaps to create some height too. But to control the width of the input, either use grid properties, or set a `width` in `%`, e.g. `width: 100%;`
If you're having trouble fitting it in, make sure it is set to `box-sizing: border-box;` as well 💡